bin/feather-icons.json

```diff
@@ -0,0 +1,259 @@
+[
+  "activity",
```

Comment:
Used some browser console JavaScript to get an array of all the possible feather icons from [the GitHub Actions documentation site](https://developer.github.com/actions/creating-github-actions/creating-a-docker-container/#supported-feather-icons).

Reply:
The `feather.json` file could be snagged from [Feather-icons](https://github.com/feathericons/feather) directly; after `npm install feather-icons` you should be able to require a big ol' JSON file as `require('feather-icons/icons.json')` though the pathing might not be right first try.

It's not _technically_ the same list that GitHub uses, but its better than having our own hard-coded list. I think that'll be more reliable, as the browser-list might change any time.

Reply:
In the same vein as opting out of including the mustache library with actions-toolkit, it seems a little heavy to include feather-icons just to get a list of the icon names in a JSON file.

Perhaps we could generate the JSON file with a small script that fetches the icon names from the feather-icons package and then writes them into this repo? Adds a little bit of maintenance overhead, but also prevents shipping an extra library that may not be used by consumers of action-toolkit (if they bypass the CLI entirely). 🤔

Reply:
Very true 🤔 Glad you picked up on my self-contradiction 😅 It _could_ grab the file from `unpkg`: https://unpkg.com/feather-icons@4.19.0/dist/icons.json - but I don't really think that's a good idea, I don't love having to depend on a 3rd party at CLI-runtime. Could be built into a little dev script I suppose 🤔

Reply:
Yeah I see pros and cons both ways.

Here is the simplest dev script I could whip up to write the feather icon names to a JSON file for use by the actions-toolkit CLI. I executed it with `node scripts/update-feather-icons.js`, and it looks like there are some differences between the GitHub docs page vs. what feather icon names exist in feather-icons@4.19.0 (as you'll see if the diff below).

```diff
diff --git a/bin/feather-icons.json b/bin/feather-icons.json
index 16ba572..07a63fc 100644
--- a/bin/feather-icons.json
+++ b/bin/feather-icons.json
@@ -53,6 +53,7 @@
   "chevrons-left",
   "chevrons-right",
   "chevrons-up",
+  "chrome",
   "circle",
   "clipboard",
   "clock",
@@ -63,6 +64,8 @@
   "cloud-snow",
   "cloud",
   "code",
+  "codepen",
+  "coffee",
   "command",
   "compass",
   "copy",
@@ -94,6 +97,7 @@
   "facebook",
   "fast-forward",
   "feather",
+  "figma",
   "file-minus",
   "file-plus",
   "file-text",
@@ -104,11 +108,14 @@
   "folder-minus",
   "folder-plus",
   "folder",
+  "frown",
   "gift",
   "git-branch",
   "git-commit",
   "git-merge",
   "git-pull-request",
+  "github",
+  "gitlab",
   "globe",
   "grid",
   "hard-drive",
@@ -120,12 +127,15 @@
   "image",
   "inbox",
   "info",
+  "instagram",
   "italic",
+  "key",
   "layers",
   "layout",
   "life-buoy",
   "link-2",
   "link",
+  "linkedin",
   "list",
   "loader",
   "lock",
@@ -136,6 +146,7 @@
   "map",
   "maximize-2",
   "maximize",
+  "meh",
   "menu",
   "message-circle",
   "message-square",
@@ -150,6 +161,7 @@
   "moon",
   "more-horizontal",
   "more-vertical",
+  "mouse-pointer",
   "move",
   "music",
   "navigation-2",
@@ -159,6 +171,7 @@
   "paperclip",
   "pause-circle",
   "pause",
+  "pen-tool",
   "percent",
   "phone-call",
   "phone-forwarded",
@@ -200,9 +213,11 @@
   "sidebar",
   "skip-back",
   "skip-forward",
+  "slack",
   "slash",
   "sliders",
   "smartphone",
+  "smile",
   "speaker",
   "square",
   "star",
@@ -221,11 +236,13 @@
   "toggle-right",
   "trash-2",
   "trash",
+  "trello",
   "trending-down",
   "trending-up",
   "triangle",
   "truck",
   "tv",
+  "twitter",
   "type",
   "umbrella",
   "underline",
@@ -250,10 +267,12 @@
   "wifi",
   "wind",
   "x-circle",
+  "x-octagon",
   "x-square",
   "x",
+  "youtube",
   "zap-off",
   "zap",
   "zoom-in",
   "zoom-out"
-]
+]
\ No newline at end of file
diff --git a/scripts/update-feather-icons.js b/scripts/update-feather-icons.js
new file mode 100644
index 0000000..03a09c7
--- /dev/null
+++ b/scripts/update-feather-icons.js
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const https = require('https')
+const path = require('path')
+
+https.get('https://unpkg.com/feather-icons@4.19.0/dist/icons.json', resp => {
+  let data = ''
+
+  resp.on('data', chunk => {
+    data += chunk
+  })
+
+  resp.on('end', () => {
+    const icons = JSON.parse(data)
+    const iconNames = Object.keys(icons)
+
+    const iconsFilePath = path.join(process.cwd(), 'bin', 'feather-icons.json')
+
+    fs.writeFile(iconsFilePath, JSON.stringify(iconNames, null, 2), err => {
+      if (err) throw err
+    })
+  })
+})
```

Reply:
Pushed up for now - see 98c36c5.